src/bitcoin-cli.cpp

```diff
@@ -235,7 +236,13 @@ UniValue CallRPC(const std::string& strMethod, const UniValue& params)
     assert(output_buffer);
     evbuffer_add(output_buffer, strRequest.data(), strRequest.size());
 
-    int r = evhttp_make_request(evcon.get(), req.get(), EVHTTP_REQ_POST, "/");
+    // check if we should use a special wallet endpoint
+    std::string endpoint = "/";
+    if (GetArg("-wallet", "") != "") {
+        // always use v1 endpoint if -wallet has been provided
+        endpoint = "/v1/wallet/"+urlEncode(GetArg("-wallet", ""));
```

Comment:
In commit "Add -wallet endpoint support to bitcoin-cli"

This doesn't compile because urlEncode function isn't defined. You should fix by moving bitcoin-cli changes from "Add urlencode/decode via libevent2" commit to this commit.